docker_mcp/models/params.py

```
@@ -0,0 +1,215 @@
+        default=None, 
+        description="Comma-separated list of hosts to select"
+    )
+    compose_path_overrides: Dict[str, str] = Field(
+        default_factory=dict, 
+        description="Per-host compose path overrides"
+    )
+    auto_confirm: bool = Field(
+        default=False, 
+        description="Auto-confirm operations without prompting"
+    )
+    cleanup_type: Optional[str] = Field(
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)